### PR TITLE
Remove NET8 if definitions

### LINKED ISSUE #17: Remove #if definitions where no longer needed for .NET 8+
Following from #16 - search for `#if NET8_0_OR_GREATER`

### PR DESCRIPTION
**Pull request by rabuckley**
Closes #17

### CODE REVIEW

File: src/DSE.Open/AsciiString.Helpers.cs [hunk 1]
```diff
@@ -84,54 +84,22 @@ public static bool SequenceEqualsCaseInsensitive(ReadOnlySpan<AsciiChar> a, Read
 
     public static void ToLowerInPlace(Span<byte> span, out int bytesWritten)
```
Comment (frankbuckley) [inline src/DSE.Open/AsciiString.Helpers.cs:85]: Should we also obsolete in favour of `System.Text.Ascii.ToLowerInPlace`?

File: src/DSE.Open/AsciiString.Helpers.cs [hunk 2]
```diff
@@ -84,54 +84,22 @@ public static bool SequenceEqualsCaseInsensitive(ReadOnlySpan<AsciiChar> a, Read
...
-#endif
-        for (var i = 0; i < span.Length; i++)
-        {
-            span[i] = AsciiChar.ToLower(span[i]);
-        }
-
-        bytesWritten = span.Length;
     }
 
     public static void ToLowerInPlace(Span<char> span, out int charsWritten)
```
Comment (frankbuckley) [inline src/DSE.Open/AsciiString.Helpers.cs:90]: Ditto: obsolete?

File: src/DSE.Open/AsciiString.Helpers.cs [hunk 3]
```diff
@@ -84,54 +84,22 @@ public static bool SequenceEqualsCaseInsensitive(ReadOnlySpan<AsciiChar> a, Read
...
-#endif
-        for (var i = 0; i < span.Length; i++)
-        {
-            span[i] = AsciiChar.ToLower(span[i]);
-        }
-
-        charsWritten = span.Length;
     }
 
     public static void ToUpperInPlace(Span<byte> span, out int bytesWritten)
```
Comment (frankbuckley) [inline src/DSE.Open/AsciiString.Helpers.cs:95]: Ditto: obsolete?

File: src/DSE.Open/AsciiString.Helpers.cs [hunk 4]
```diff
@@ -84,54 +84,22 @@ public static bool SequenceEqualsCaseInsensitive(ReadOnlySpan<AsciiChar> a, Read
...
-#endif
-        for (var i = 0; i < span.Length; i++)
-        {
-            span[i] = AsciiChar.ToUpper(span[i]);
-        }
-
-        bytesWritten = span.Length;
     }
 
     public static void ToUpperInPlace(Span<char> span, out int charsWritten)
```
Comment (frankbuckley) [inline src/DSE.Open/AsciiString.Helpers.cs:100]: Ditto: obsolete?

**Review by frankbuckley — APPROVED** [review]

**Comment by rabuckley** [comment]
@frankbuckley they are not used, so went ahead and removed.

**Comment by frankbuckley** [comment]
> @frankbuckley they are not used, so went ahead and removed.

That will/may be breaking downstream...

**Comment by rabuckley** [comment]
> That will/may be breaking downstream...

Where?

**Comment by frankbuckley** [comment]
> > That will/may be breaking downstream...
> 
> Where?

hve you checked?

**Comment by rabuckley** [comment]
Yes, I meant "they are not used" anywhere, not just in this solution.

**Pull request merged**